doc/source/rllib-algorithms.rst

```diff
@@ -399,15 +402,23 @@ HalfCheetah    9664                       ~7700
 Soft Actor Critic (SAC)
 ------------------------
 |pytorch| |tensorflow|
-`[paper] <https://arxiv.org/pdf/1801.01290>`__ `[implementation] <https://github.com/ray-project/ray/blob/master/rllib/agents/sac/sac.py>`__
+`[original paper] <https://arxiv.org/pdf/1801.01290>`__,
+`[follow up paper] <https://arxiv.org/pdf/1812.05905.pdf>`__,
```

Comment:
Perhaps put all papers and follow ups on one line

Reply:
done